### PR TITLE
[release] framework release of v1.8

### PR DESCRIPTION
**Pull request by junkil-park**

### CODE REVIEW

File: metadata/v1.8/step_1_upgrade_framework.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.8",
+  "description": "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.8",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```
Comment (sherry-x) [inline metadata/v1.8/step_1_upgrade_framework.json:4]: source code url here should be the source files you created together with this PR, but we can use another PR to update them

**Review by sherry-x — APPROVED** [review]

File: metadata/v1.8/step_1_upgrade_framework.json (hunk 1 above)
Comment (junkil-park) [reply metadata/v1.8/step_1_upgrade_framework.json]: By the way, this `source_code_url` seems to be auto-generated. `release.yaml` did have the field.

Comment (junkil-park) [reply metadata/v1.8/step_1_upgrade_framework.json]: I think the previous `source_code_url` was the default string. The original `release.yaml` didn't have the field.

**Review by movekevin — APPROVED** [review]

File: metadata/v1.8/step_1_upgrade_framework.json (hunk 1 above)
Comment (junkil-park) [reply metadata/v1.8/step_1_upgrade_framework.json]: @sherry-x , I updated `source_code_url` to https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/v1.8/step_1_upgrade_framework

**Comment by sherry-x** [comment]
LGTM!

**Pull request merged**